### ISSUE TITLE
Disable craftable pets?

### ISSUE THREAD

**Issue by Gusbars** [capnkirok/Inventory-Pets#434]
I would like to disable the ability for players to craft pets.
I'm running Minecraft 1.16.5 
and the mod version is: 2.0.12

Thank you!
Gustavo

**Comment by capnkirok** [comment]
Me too @Gusbars - I still haven't figured out how to do this in 1.15/1.16. I'm going to do some research and see if any other modders have figured this one out yet. Leaving on the board as a reminder.

**Comment by Krazyman999** [comment]
+1 to No Crafting Pets Config Option

**Comment by yasso69** [comment]
yeah me too add an option in config to disable craftind them

**Comment by Krazyman999** [comment]
I may have fixed it so you can't craft them, but I've only done some basic testing on my own unpublished modpack.
.
.
I opened the Zip file, I did this on my CurseForge modpack. Then removed the Recipes manually, from the Zip file.
.. They can't be crafted, but I haven't tested it fully... Soo... USE AT YOUR OWN RISK...

1. Open inventorypets Jar File, it can be already installed in your CurseForge Modpack.
2. Click on Data
3. Click on inventorypets
4. Click on Recipes
5. You need to delete only the pets, leaving the Feeding Bag, and the Pet Foods.
.
.
Don't delete...

coal
diamond
feed_bag
lapis_lazuli
nugget_coal
nugget_coal_alt
nugget_diamond
nugget_diamond_alt
nugget_emerald
nugget_emerald_alt
nugget_ender
nugget_ender_alt
nugget_lapis_alt
nugget_obsidian
nugget_obsidian_alt
obsidian

**Comment by capnkirok** [comment]
Well, I could actually publish an alternate version on CurseForge with recipes removed, that never even occurred to me. However, I had an idea recently I want to try out. Stay tuned.

**Comment by capnkirok** [comment]
You can also use CraftTweaker mod to disable/modify recipes.

**Comment by Krazyman999** [comment]
Yeah... I used Craft Tweaker for my Shipwrecks mod Diving Suit patch, on Planet Minecraft.

Half the fun of your mod, is trying to find and get the items, without dying. A version without crafting the pets, would be nice... :)

**Comment by capnkirok** [comment]
There used to be a handy event where you could just remove any recipes prior to load, but it was removed after 1.15x. Now what used to be very simple is now quite complex. Still looking into this.

**Comment by capnkirok** [comment]
> I may have fixed it so you can't craft them, but I've only done some basic testing on my own unpublished modpack. . . I opened the Zip file, I did this on my CurseForge modpack. Then removed the Recipes manually, from the Zip file. .. They can't be crafted, but I haven't tested it fully... Soo... USE AT YOUR OWN RISK...
> 
>     1. Open inventorypets Jar File, it can be already installed in your CurseForge Modpack.
> 
>     2. Click on Data
> 
>     3. Click on inventorypets
> 
>     4. Click on Recipes
> 
>     5. You need to delete only the pets, leaving the Feeding Bag, and the Pet Foods.
>        .
>        .
>        Don't delete...
> 
> 
> coal diamond feed_bag lapis_lazuli nugget_coal nugget_coal_alt nugget_diamond nugget_diamond_alt nugget_emerald nugget_emerald_alt nugget_ender nugget_ender_alt nugget_lapis_alt nugget_obsidian nugget_obsidian_alt obsidian

I approve of this method. Simple to do... just rename the file extension to .zip and then you can go and remove the recipes as @Krazyman999 recommends.
